osf/management/commands/sync_citation_styles.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand
+import re
+import os
+import io
+import zipfile
+import requests
+import logging
+from django.apps import apps
+from api.base import settings
+
+from django.db import transaction
+from lxml import etree
+from urllib.parse import urlparse
+
+logger = logging.getLogger(__name__)
+
+
+def sync_citation_styles(dry_run=False):
+    CitationStyle = apps.get_model('osf', 'citationstyle')
+    zip_data = io.BytesIO(requests.get(settings.CITATION_STYLES_REPO_URL).content)
```

Comment:
* Why loading data from an online resource now? 
  * In comparison, the old function loads from files. `for style_file in get_style_files(settings.CITATION_STYLES_PATH):`
* Related, why do you choose this file: 
  * https://github.com/CenterForOpenScience/styles/archive/88e6ed31a91e9f5a480b486029cda97b535935d4.zip
  * My understanding is that we run this command when we have citation styles to update. Thus the input should not be hard-coded but instead be provided as an option.

Reply:
It's actually a weird logically thing because if you look at the content of `CITATION_STYLES_PATH` it contains the files from the repo at `CITATION_STYLES_REPO_URL`, so the files at `CITATION_STYLES_PATH` are just being loaded from an external resource anyway, just via that `bower install` statement I removed, without bower installing the files at `CITATION_STYLES_PATH` I had to download the repo via requests. 

I picked that specific url because it's pegged to a specific commit. 

That's a good point about pass the url as parameter, but I want to be able to save the current url in the settings where I feel it's easier to find.